static/app/views/organizationStats/index.tsx

```diff
@@ -394,22 +394,25 @@ export class OrganizationStats extends Component<Props> {
                   />
                 </ErrorBoundary>
               </div>
-              {!isSingleProject && (
-                <ErrorBoundary mini>
-                  <UsageStatsProjects
-                    organization={organization}
-                    dataCategory={this.dataCategory}
-                    dataCategoryName={this.dataCategoryName}
-                    projectIds={this.projectIds}
-                    dataDatetime={this.dataDatetime}
-                    tableSort={this.tableSort}
-                    tableQuery={this.tableQuery}
-                    tableCursor={this.tableCursor}
-                    handleChangeState={this.setStateOnUrl}
-                    getNextLocations={this.getNextLocations}
-                  />
-                </ErrorBoundary>
-              )}
+              <ErrorBoundary mini>
+                {isSingleProject && (
+                  <PanelHeading>
+                    <Title>{t('All Projects')}</Title>
+                  </PanelHeading>
+                )}
+                <UsageStatsProjects
+                  organization={organization}
+                  dataCategory={this.dataCategory}
+                  dataCategoryName={this.dataCategoryName}
+                  projectIds={isSingleProject ? [ALL_ACCESS_PROJECTS] : this.projectIds}
```

Comment:
Uses the selected projectIds unless you've only chosen a single project, then it shows all projects